.github/workflows/safety.yml

```diff
@@ -0,0 +1,42 @@
+name: Safety
```

Comment:
Er dette et problem som kommer til å treffe flere repoet, slik at denne workflowen bør være i workflow repoet?

Reply:
for andre repoer så kommer vi til å brukde den generelle versjonen som hans olav har pr på i workflows-repoet